merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/http/ResponseSerializers.java

```diff
@@ -191,6 +191,14 @@ public static JsonValue serializeBulkArgumentValidationResponse(BulkArgumentVali
       return Json.createObjectBuilder(serializeInstantiationException(f.ex()).asJsonObject())
           .add("type", "INSTANTIATION_ERRORS")
           .build();
+    } else if (response instanceof BulkArgumentValidationResponse.NoSuchMissionModelError m) {
+      return Json.createObjectBuilder()
+                 .add("errors", Json.createObjectBuilder()
+                     .add("noSuchMissionModelError", serializeNoSuchMissionModelException(m.ex()))
+                     .build())
+                 .add("success", JsonValue.FALSE)
+                 .add("type", "NO_SUCH_MISSION_MODEL")
+                 .build();
```

Comment:
Extremely minor, nonblocking: could you reorder this so the keys are added in the same order as in the other responses (ie `Validation`, `NoSuchActivityError`)?

Reply:
Change has been made!